flask/app/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.7-alpine 
+WORKDIR /app 
+COPY . /app 
+RUN pip3 install -r requirements.txt 
+ENTRYPOINT ["python3"] 
+CMD ["app.py"]
```

Comment:
nit: add a new line at the end of the file

Reply:
Done